HHS/OPRE-OPS
# feat: ✨adds CAN Funding Cards to CAN Funding page

## What changed

- adds CAN Funding Cards to the CAN Funding tab
- could not find data for Funding Received, Funding Method, and Funding Type

## Issue

#2673 #3080 

## How to test

1. e2e tests pass
2. or goto `/cans/504/funding`
3. set fy to 2024
4. see cards light up
5. 💰 profit

## Screenshots

<img width="988" alt="image" src="https://github.com/user-attachments/assets/c90ab9d6-3c37-47a9-8710-fff3e20b7275">

## Definition of Done Checklist
